### ISSUE TITLE
NoClassDefFoundError: org/bouncycastle/asn1/DERBoolean when using jbundler with trinidad

### ISSUE THREAD

**Issue by griff** [mkristian/jbundler#57]
I am getting some kind of bad interaction between jruby, trinidad and jbundler which means whenever I try to start trinidad I get a NoClassDefFoundError: org/bouncycastle/asn1/DERBoolean

After having spent several hours trying to figure it out myself I am a bit at a loss as to what the problem is. But I have managed to reduce the problem so that I can reproduce it with this gist: https://gist.github.com/griff/935a5f1f903c0455d1ee

I can say that if there is no Jarfile or I don't require jbundler I don't get the error so I am pretty sure that it is a jbundler problem.

**Comment by mkristian** [comment]
yes there is some problem with the jars from default dependencies :( and
jbundler

you can add
```
gem 'jruby-openssl', '0.9.5'
```
to your Gemfile which works around this problem (or take version 0.9.6 the
latest jruby-openssl gem)

**Comment by mkristian** [comment]
jbundler-0.8.0 should fix your problem.
